docs/changehistory/NextVersion.md

```diff
@@ -89,6 +89,10 @@ The AuthStatus enum has been removed. This enum has fallen out of use since the
 
 The beta functions [Element.collectPredecessorIds]($core-backend) and [Element.getPredecessorIds]($core-backend) have been deprecated and replaced with [Element.collectReferenceIds]($core-backend) and [Element.getReferenceIds]($core-backend), since the term "predecessor" has been inaccurate since 3.2.0, when the transformer became capable of handling cyclic references and not just references to elements that were inserted before itself (predecessors).
 
+### @itwin/core-geometry
+
+The [GrowableXYArray]($core-geometry) method `setXYZAtCheckedPointIndex` is deprecated in favor of the more appropriately named new method `setXYAtCheckedPointIndex`.
+
```

Comment:
I think it's worth calling out the performance improvements here... no need for numbers since they'd be so benchmark-specific, but we know this will always be at least marginally faster than the previous version.

Reply:
Good idea!  Will do.